[2019-12-17 10:33:26,195]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:23} WARNING-REQUEST_LOG	{"request": {}, "response": [{"id": 1, "name": "user ke 1", "age": 15, "sex": "Male", "client_id": 51}, {"id": 2, "name": "user ke 2", "age": 26, "sex": "Male", "client_id": 36}, {"id": 3, "name": "user ke 3", "age": 47, "sex": "Male", "client_id": 57}, {"id": 4, "name": "user ke 4", "age": 23, "sex": "Female", "client_id": 38}]}
[2019-12-17 10:33:47,922]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:27} WARNING-REQUEST_LOG	{"request": {"age": 15, "sex": "Male", "client_id": 51}, "response": {"message": {"name": "Missing required parameter in the JSON body"}}}
[2019-12-17 10:36:06,370]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:23} WARNING-REQUEST_LOG	{"request": {}, "response": [{"id": 1, "name": "user ke 1", "age": 15, "sex": "Male", "client_id": 51}, {"id": 2, "name": "user ke 2", "age": 26, "sex": "Male", "client_id": 36}, {"id": 3, "name": "user ke 3", "age": 47, "sex": "Male", "client_id": 57}, {"id": 4, "name": "user ke 4", "age": 23, "sex": "Female", "client_id": 38}]}
[2019-12-17 10:54:23,566]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:43} ERROR-REQUEST_LOG	{"status_code": 400, "method": "POST", "code": "400 BAD REQUEST", "uri": "/user?", "request": {}, "response": {"message": "Failed to decode JSON object: Expecting value: line 1 column 1 (char 0)"}}
[2019-12-17 11:01:28,894]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:43} ERROR-REQUEST_LOG	{"status_code": 400, "method": "POST", "code": "400 BAD REQUEST", "uri": "/user?", "request": {}, "response": {"message": "Failed to decode JSON object: Expecting value: line 1 column 1 (char 0)"}}
[2019-12-17 11:08:37,174]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:32} INFO-REQUEST_LOG	{"status_code": 200, "method": "GET", "code": "200 OK", "uri": "/user?", "request": {}, "response": [{"id": 1, "name": "user ke 1", "age": 82, "sex": "Female", "client_id": 22}, {"id": 2, "name": "user ke 2", "age": 73, "sex": "Female", "client_id": 10}, {"id": 3, "name": "user ke 3", "age": 17, "sex": "Female", "client_id": 28}, {"id": 4, "name": "user ke 4", "age": 3, "sex": "Male", "client_id": 13}]}
[2019-12-17 11:08:57,678]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:32} INFO-REQUEST_LOG	{"status_code": 200, "method": "GET", "code": "200 OK", "uri": "/user/20?", "request": {}, "response": {"status": "NOT_FOUND"}}
[2019-12-17 11:09:25,855]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:32} INFO-REQUEST_LOG	{"status_code": 200, "method": "GET", "code": "200 OK", "uri": "/user?", "request": {}, "response": [{"id": 1, "name": "user ke 1", "age": 82, "sex": "Female", "client_id": 22}, {"id": 2, "name": "user ke 2", "age": 73, "sex": "Female", "client_id": 10}, {"id": 3, "name": "user ke 3", "age": 17, "sex": "Female", "client_id": 28}, {"id": 4, "name": "user ke 4", "age": 3, "sex": "Male", "client_id": 13}]}
[2019-12-17 11:10:42,863]{/home/alta9/Documents/Alta Batch 4/Phase 2/Week 1/Day 1/Challenge_2/app.py:43} ERROR-REQUEST_LOG	{"status_code": 404, "method": "GET", "code": "404 NOT FOUND", "uri": "/user/20?", "request": {}, "response": {"status": "NOT_FOUND", "message": "Client not found"}}
[2019-12-17 12:20:00,937]{/home/alta9/Documents/Alta4_Hamdi/fullstack4_hamdi/REST API/Challenge_3/app.py:43} ERROR-REQUEST_LOG	{"status_code": 501, "method": "PATCH", "code": "501 NOT IMPLEMENTED", "uri": "/user?", "request": {}, "response": "Not yet implement"}
[2019-12-17 12:24:14,341]{/home/alta9/Documents/Alta4_Hamdi/fullstack4_hamdi/REST API/Challenge_3/app.py:54} WARNING-REQUEST_LOG	{"status_code": 400, "method": "POST", "code": "400 BAD REQUEST", "uri": "/user?", "request": {}, "response": {"message": "Failed to decode JSON object: Expecting value: line 1 column 1 (char 0)"}}
[2019-12-17 12:25:13,743]{/home/alta9/Documents/Alta4_Hamdi/fullstack4_hamdi/REST API/Challenge_3/app.py:54} WARNING-REQUEST_LOG	{"status_code": 404, "method": "POST", "code": "404 NOT FOUND", "uri": "/use?", "request": {}, "response": {"message": "The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again."}}
[2019-12-17 12:25:42,991]{/home/alta9/Documents/Alta4_Hamdi/fullstack4_hamdi/REST API/Challenge_3/app.py:32} INFO-REQUEST_LOG	{"status_code": 200, "method": "GET", "code": "200 OK", "uri": "/user?", "request": {}, "response": [{"id": 1, "name": "user ke 1", "age": 32, "sex": "Female", "client_id": 11}, {"id": 2, "name": "user ke 2", "age": 32, "sex": "Male", "client_id": 4}, {"id": 3, "name": "user ke 3", "age": 54, "sex": "Female", "client_id": 15}, {"id": 4, "name": "user ke 4", "age": 13, "sex": "Male", "client_id": 41}]}
